Doc/includes/email-dir.py

```diff
@@ -39,9 +39,10 @@ def main():
     directory = args.directory
     if not directory:
         directory = '.'
+    context = os.path.abspath(directory)
     # Create the message
     msg = EmailMessage()
-    msg['Subject'] = 'Contents of directory %s' % os.path.abspath(directory)
+    msg['Subject'] = f'Contents of directory {context}'
```

Comment:
While I don't have an issue with the changes made (other than the name of the variable perhaps, I'd prefer "path" instead of "context"), it's not related to changing the examples to use f-strings. Also, in this case, I don't think using a variable here makes the example particularly more clear or readable. 

For simplifying the review process, this change to the example should probably be omitted from the PR:
```suggestion
    msg['Subject'] = f'Contents of directory {os.path.abspath(directory)}'
```

(in addition to removing `context = os.path.abspath(directory)` on line 42 of course)